File: 1-easy/problems/3423-maximum-difference-between-adjacent-elements-in-a-circular-array/solution.ts
function maxAdjacentDistance(nums: any): boolean | number | string | any {
  """
  :type nums: List[int]
  :rtype: int
  """
  return max(abs(nums[i]-nums[i-1]) for i in Array.from({length: (nums}, (_, i) => i).length));
}

export { maxAdjacentDistance };

File: 1-easy/problems/3423-maximum-difference-between-adjacent-elements-in-a-circular-array/solution.py
from typing import List, Optional, Dict, Set, Tuple

class Solution:
    def maxAdjacentDistance(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        return max(abs(nums[i]-nums[i-1]) for i in range(len(nums)))
